### PR TITLE
Roughness ECMA 418-2

### PR DESCRIPTION
**Pull request by wantysal**
New version of roughness calculation according to ECMA 418-2.
The code uses Sottek's Hearing Model implemented in loudness_ecma. The final roughness values are not validated yet due to some weighting errors.

### CODE REVIEW

**Comment by wantysal** [comment]
Thanks to a collaboration with @fchirono the code has been updated and the model now seems very close to the ECMA 418-2 results. Validation plots have been created :
roughness_validation
And all tests are running:
image

**Comment by fchirono** [comment]
Excellent work Salomé, the new validation results look great! 😃

**Review by mglesser — CHANGES_REQUESTED** [review]
I made 2 minor comments directly in the files. I'm also wondering if we shouldn't take the occasion of this release to remove the tutorials that are probably outdated now. And maybe also to simmplify the main readme with a link to the documentation website.

File: mosqito/sq_metrics/roughness/roughness_ecma/_von_hann_window.py
[the remark this answers is not in the document]
Comment (fchirono) [reply mosqito/sq_metrics/roughness/roughness_ecma/_von_hann_window.py:1]: The ECMA 418-2 standard specifies a normalised version of the Hann window, with the extra `sqrt(0.375)` factor when compared to the SciPy (or identical Numpy) version. My personal opinion is that it is worth having it as a separate, explicit function to emphasize it differs from the usual definition, but this is not something I feel too strongly about it - I'm happy either way.

**Comment by wantysal** [comment]
Hi @mglesser,

Requested changes have been made ;)

Tks for your review

**Pull request merged**